tests/integration/defs/perf/pytorch_model_config.py

```
@@ -19,6 +19,30 @@
+
+
+def _get_sm_version_safe() -> int:
+    """Return the current device SM version, or 0 if it cannot be determined.
+
+    Imported lazily so importing this module does not require CUDA to be
+    available (e.g. during static test collection).
+    """
+    try:
+        from tensorrt_llm._utils import get_sm_version
+        return get_sm_version()
+    except Exception:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)